site/docs/relations/logical_relations.md

```diff
@@ -293,11 +293,11 @@ The fetch operation eliminates records outside a desired window. Typically corre
 
 ### Fetch Properties
 
-| Property | Description                                                  | Required                 |
-| -------- | ------------------------------------------------------------ | ------------------------ |
-| Input    | A relational input, typically with a desired orderedness property. | Required                 |
-| Offset   | A positive integer. Declares the offset for retrieval of records. | Optional, defaults to 0. |
-| Count    | A positive integer. Declares the number of records that should be returned. | Required                 |
+| Property | Description                                                           | Required                 |
+| -------- | --------------------------------------------------------------------- | ------------------------ |
+| Input    | A relational input, typically with a desired orderedness property.    | Required                 |
+| Offset   | A non-negative integer. Declares the offset for retrieval of records. | Optional, defaults to 0. |
+| Count    | A non-negative integer or -1. Declares the number of records that should be returned. -1 signals that ALL records should be returned. | Required |
```

Comment:
We could also allow zero for testing purposes (as mentioned on the issues thread) but we can also address that outside of this PR.

Reply:
My intent with changing these to `non-negative integer` was to make it explicit that 0 _was_ allowed. Should I update the wording if this is not clear?

Reply:
I missed that distinction.  Yes, this is fine.